README.rst

```diff
@@ -1,6 +1,18 @@
-|Tests Status| |Coverage| |Gitter|
+|Tests Status|
+|Coverage|
+|Gitter|
 
 
+.. |Tests Status| image:: https://github.com/aesara-devs/aeppl/actions/workflows/test.yml/badge.svg?branch=main
+    :target: https://github.com/aesara-devs/aeppl/actions/workflows/test.yml
+
+.. |Coverage| image:: https://codecov.io/gh/aesara-devs/aeppl/branch/main/graph/badge.svg?token=L2i59LsFc0
+    :target: https://codecov.io/gh/aesara-devs/aeppl
+
+.. |Gitter| image:: https://badges.gitter.im/aesara-devs/aeppl.svg
+   :alt: Join the chat at https://gitter.im/aesara-devs/aeppl
+   :target: https://gitter.im/aesara-devs/aeppl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
+
```

Comment:
Why are these being moved?

Reply:
we has some issues with the badge rendering and tried moving that to see if it helped, then accidentally committed the change after fixing the actual issue.